src/api/dlp/BorrowerStandardPortalGeneralSettingsApi.ts

```diff
@@ -3,11 +3,16 @@ import { BorrowerType } from '../../enums';
 import DigitalLendingPortalEnvironment from '../../enums/DigitalLendingPortalEnvironment';
 import getSearchParams from '../../utils/getSearchParams';
 
-export interface BorrowerStandardPortalGeneralSettings {
+export interface StandardBorrowerPortalConfig {
```

Comment:
is this interface was implemented in prior version? (not in alpha)

Reply:
no, it's a new one

Reply:
i mean, BorrowerStandardPortalGeneralSettings

Reply:
BorrowerStandardPortalGeneralSettings existed before

Reply:
but with different properties